packages/components/psammead-copyright/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.4.0   | [PR#617](https://github.com/bbc/psammead/pull/617) Add optional `position` prop, and default position to left of its parent container |
```

Comment:
Since we're using this in production, we should really bump this to be v1.0.0

Reply:
Gotcha